src/System.Private.ServiceModel/tools/scripts/WCFCustomCleanup.cmd

```diff
@@ -0,0 +1,36 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+REM
+echo Cleanup test services
+REM  Note that we always return successful code in clean up routine because clean up errors are not fatal
+
+TASKKILL /F /IM SelfHostedWCFService.exe
+REM error 128 will be returned if The process not found
+if NOT [%ERRORLEVEL%]==[0]  if NOT [%ERRORLEVEL%]==[128] (
+	echo Warning: An error occurred while killing the SelfHostedWCFService.exe.
+	)
+
+call %~dp0CleanUpHttpsPort.cmd
+if NOT [%ERRORLEVEL%]==[0] (
+	echo Warning: An error occurred while removing https port.
+	)
+
+call %~dp0RemoveFirewallPorts.cmd
+if NOT [%ERRORLEVEL%]==[0] (
+	echo Warning: An error occurred while removing Firewall ports.
+	)
+
+call %~dp0..\..\..\..\bin\Wcf\tools\CertificateGenerator\CertificateGenerator.exe -Uninstall
```

Comment:
Might be worth checking if this exists first before trying to run it; if someone did a partial build of the WCF repo then it's possible that the CertificateGenerator has not been created yet.